website_snippet_anchor/README.rst

```diff
@@ -0,0 +1,80 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Set Snippet's Anchor
+====================
+
+This module extends the functionality of the website editor to support setting
+an anchor to any section of your web page and allow you to link to them very
+easily.
+
+Anchors are just the HTML ``id`` attribute, which is commonly used to add it
+to any URL and get directly to the element with that anchor, in the form of
+http://www.antiun.com/#anchor-name.
+
+Usage
+=====
+
+To add an anchor to any element, you need to:
+
+* Edit any web page.
+* Go to *Insert blocks > Structure*.
```

Comment:
First paragraph says "to any section of your web page". I would not consider a section something that is not inside structure menu.